experimental/traffic-portal/src/app/shared/navigation/tp-sidebar/tp-sidebar.component.scss

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+mat-sidenav-container {
+	height: calc(100% - 3.9em);
+
+	mat-sidenav {
+		min-width: 10em;
+
+		.mat-icon {
+			margin-right: 0;
+		}
+
+
+		div.bottom {
+			position: absolute;
+			bottom: 0;
+			left: 0;
+			width: 100%;
+		}
+
+		.mat-menu-item {
+			width: 100%;
```

Comment:
This width setting is causing a horizontal scrollbar to appear on the opened sidebar